src/components/Certify/Certify.vue

```diff
@@ -125,7 +122,7 @@ export default class Certify extends Vue {
 }
 
 @media (min-width: 768px) {
-  .certifiedby-container {
+  .certifiedBy-container {
```

Comment:
Should be "certified-by-container".

Reply:
copy that

Reply:
I looked it up for fun...
`camelCase`
`PascalCase` aka `TitleCase`
`snake_case`
`kebab-case`

Reply:
I don't think there's such as thing as Camel Kebabs 🤣

Reply:
hahah well there is now!! 
😆 Jk jK.